catalog/app/containers/StoryBook/JsonEditor.tsx

```diff
@@ -8,16 +8,18 @@ import { JsonValue, ValidationErrors } from 'components/JsonEditor/constants'
 const schema = {
   type: 'object',
   properties: {
-    a: { type: 'number' },
+    a: { anyOf: [{ type: 'number', minimum: 1024 }, { type: 'string' }] },
     b: { type: 'number' },
   },
 }
 const validate = jsonSchema.makeSchemaValidator(schema)
+const noop = () => {}
 
 export default function JsonEditorBook() {
   const [value, setValue] = React.useState<JsonValue>({
-    a: '123',
-    b: 123,
+    a: 123,
```

Comment:
is changing a string into a number intentional?

Reply:
Yes. To make object invalid